backend/src/v5/processors/teamspaces/projects/models/drawings.js

```diff
@@ -40,16 +42,19 @@ Drawings.getDrawingList = async (teamspace, project, user) => {
 
 Drawings.getDrawingStats = async (teamspace, drawing) => {
 	let latestRev;
+	let calibration;
+
 	const [settings, revCount] = await Promise.all([
-		getDrawingById(teamspace, drawing, { number: 1, status: 1, type: 1, desc: 1, calibration: 1 }),
+		getDrawingById(teamspace, drawing, { number: 1, status: 1, type: 1, desc: 1 }),
 		getRevisionCount(teamspace, drawing, modelTypes.DRAWING),
 	]);
 
 	try {
```

Comment:
put this function in a helper function and have that function called as part of the Promise.all above?
I don't think there is a dependency with the 2 functions in that array right?